cloud-elements/Cloud Elements (Demo).json

```diff
@@ -0,0 +1 @@
_id":{"x-samplevalue":173,"format":"int32","type":"integer"},"company_name":{"x-samplevalue":"caleb geene's  Company","type":"string"},"element_api":{"x-samplevalue":"avadaKedavra","type":"string"},"company_external_name":{"x-samplevalue":"NA","type":"string"},"element_name":{"x-samplevalue":"Cloud Elements","type":"string"},"request_id":{"x-samplevalue":"5b31352ae4b089f56fbae41d","type":"string"}}}}},"type":"api","hooks":[{"isLegacy":false,"mimeType":"application/javascript","body":"let parameters = request_vendor_parameters\n\nif (request_expression) {\n \t for (let i = 0; i < request_expression.length; i++) {\n   \t\tif (request_expression[i].operator === \">\" || request_expression[i].operator === \">=\") {\n       \t\tparameters.from = request_expression[i].value; \n      } else {\n        \tparameters.to = request_expression[i].value;\n      }\t\n   }\n}\n\ndelete parameters.where\ndelete parameters.page\ndelete parameters.pageSize\ndelete parameters.pageOffset\n\ndone({\n\trequest_vendor_parameters: parameters,\n  continue: true\n})","type":"preRequest"},{"isLegacy":false,"mimeType":"application/javascript","body":"var response = response_body;\nvar responseStatusCode = response_status_code;\nif(responseStatusCode == 404) {\n\tvar message = response.message;\n  if(message == \"No usage logs found\") {\n  \tresponseStatusCode = 200;\n    response = [];\n  }\n} else {\n\tfor (let i = 0; i < response.length; i++) {\n  \tresponse[i]['id'] = response[i]['request_id'];\n  }\n}\ndone({\n\t\"response_status_code\": responseStatusCode,\n  \"response_body\": response\n})","type":"postRequest"}],"modelMigrated":false,"parameters":[{"vendorType":"no-op","dataType":"string","name":"where","description":"The CEQL search expression","vendorDataType":"string","source":"request","type":"query","vendorName":"where","required":false},{"vendorType":"query","dataType":"string","name":"page","description":"The page number of resources to retrieve","vendorDataType":"string","source":"request","type":"query","vendorName":"pageOffset","required":false},{"vendorType":"query","dataType":"string","name":"pageSize","description":"The number of resources to return in a given page","vendorDataType":"string","source":"request","type":"query","vendorName":"pageSize","required":false}],"vendorPath":"/usage"}],"protocolType":"http","transformationsEnabled":true,"deleted":false,"hub":"general","typeOauth":false,"name":"Cloud Elements (Demo)","extendable":true,"parameters":[{"vendorType":"header","name":"application/json","source":"request","type":"value","vendorName":"Accept","required":false},{"vendorType":"header","name":"auth.header","source":"request","type":"configuration","vendorName":"Authorization","required":false},{"vendorType":"header","name":"application/json","source":"request","type":"value","vendorName":"Content-Type","required":false},{"vendorType":"path","name":"subdomain","source":"request","type":"configuration","vendorName":"subdomain","required":false}]}
```

Comment:
This seems frighteningly specific:
```
                            "headers": {
                                "type": "string",
                                "x-samplevalue": "referer: https://my-staging.cloudelements.io/elements/11607/resources, x-forwarded-proto: https, cookie: _ga=GA1.2.1635220503.1522040780; _biz_uid=17d1c3250552497bd346bbd5d3dd5348; hubspotutk=8214706479cbdfd9bfc22d3a097c3f86; __qca=P0-403071902-1522040780505; _biz_flagsA=%7B%22Version%22%3A1%2C%22XDomain%22%3A%221%22%7D; intercom-id-pq50auuw=2da3558e-cf6c-4c37-ab68-5aaee16a4f51; _ceir=1; _CEFT=EgNwlgpg7hAmBcBHALgOwFICUCMBaAtgFoAOAzgNACq-AnAF7L4AaA0gJIAsbAQGLAAh0A1gCtSiAPZtsAdQDKmAOwTiAJgDm8AMYTUICACdkcJGix4iZeNg4AGLQDZFW27nsBmd7g4BDdwCNcGgAOYIAzIP8w2wBWHw4Y_3cnAEBsIA; ei_client_id=5b16137f345c2c0016dc7c5d; uf_privacy_prefs=1%7C1; _ceg.s=pdft63; _ceg.u=pdft63; __ar_v4=%7CCYAENERYQVDRFJAONGBDWI%3A20180813%3A1%7CDB3LZGJDU5CJFPLZ2AHAWB%3A20180813%3A1%7CR3RAPKTUGBFBDMK56QD6LW%3A20180813%3A1; _gid=GA1.2.1343923056.1539070939; _biz_nA=20; _biz_pendingA=%5B%5D; __hssrc=1; __hstc=241474653.8214706479cbdfd9bfc22d3a097c3f86.1522040780406.1539070944590.1539149946220.32; _hp2_id.4192609321=%7B%22userId%22%3A%228496127277145776%22%2C%22pageviewId%22%3A%225703318027042801%22%2C%22sessionId%22%3A%223177849872962193%22%2C%22identity%22%3Anull%2C%22trackerVersion%22%3A%224.0%22%7D, origin: https://my-staging.cloudelements.io, x-consumer-id: d36776d3-42e4-438f-a5bb-ab9a2e1f3b6a, x-forwarded-port: 443, x-real-ip: 127.0.0.1, x-amzn-trace-id: Root=1-5bbda490-aaf774bc9672370456c1652c, x-forwarded-host: loopback, x-consumer-username: ibm@ce.com, content-type: application/json, x-consumer-custom-id: 884:3840:3604"
```
Could you genericize it?

Reply:
@rrothenb - could you help point me to where this is coming from/why it's included in such a specific way in the swagger? agreed it's frighteningly specific, i'm just not sure the best way to genericize it...

Reply:
@carlyfrederick - Good question.  Since it's a custom field (`x-`), I'm guessing that EB adds this (poorly).  I think the best way to deal with this is to just remove it from the JSON.  Thoughts?

Reply:
i'm fine with that - would removing the entire `headers` object break anything or should i remove the entire parent object starting on line 466 with `"path":"/hubs/general/usage"`?

Reply:
You should keep `headers` but remove `x-samplevalue`, which is basically ignored by swagger anyway.

Reply:
should be all set!